[2024-01-02T21:18:26.945+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-04-06T00:00:00+00:00 [queued]>
[2024-01-02T21:18:26.952+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-04-06T00:00:00+00:00 [queued]>
[2024-01-02T21:18:26.953+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2024-01-02T21:18:26.964+0100] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_into_table> on 2023-04-06 00:00:00+00:00
[2024-01-02T21:18:26.967+0100] {standard_task_runner.py:57} INFO - Started process 85632 to run task
[2024-01-02T21:18:26.973+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_into_table', 'scheduled__2023-04-06T00:00:00+00:00', '--job-id', '14858', '--raw', '--subdir', '/Users/user/Desktop/Learning/AIRFLOW-TUTORIAL/methode-1/dags/dag_with_postgres.py', '--cfg-path', '/var/folders/gn/cr4yj18d1tv34pkws9v_kq1r0000gn/T/tmpi9vt06s9']
[2024-01-02T21:18:26.976+0100] {standard_task_runner.py:85} INFO - Job 14858: Subtask delete_into_table
[2024-01-02T21:18:27.021+0100] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-04-06T00:00:00+00:00 [running]> on host macbook-pro-de-user.local
[2024-01-02T21:18:27.469+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-04-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-06T00:00:00+00:00'
[2024-01-02T21:18:27.470+0100] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt='2023-04-06' and dag_id='dag_with_postgres_operator_v04'
        
[2024-01-02T21:18:27.475+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T21:18:27.642+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T21:18:27.679+0100] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt='2023-04-06' and dag_id='dag_with_postgres_operator_v04'
        , parameters: None
[2024-01-02T21:18:27.683+0100] {sql.py:427} INFO - Rows affected: 0
[2024-01-02T21:18:27.688+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_into_table, execution_date=20230406T000000, start_date=20240102T201826, end_date=20240102T201827
[2024-01-02T21:18:27.744+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:18:27.765+0100] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
